locales/en/messages.json

```diff
@@ -3490,17 +3490,17 @@
         "message": "LEDs without wire ordering number will not be saved.",
         "description": "Message in the wiring modes in Led Strip"
     },
-    "ledStripVtxFunction": {
+    "ledStripLarsonOverlay": {
         "message": "Larson scanner"
     },
```

Comment:
Any change in the messages files will force the translators to traslate them again. I don't use leds, but I suppose the change is because it describes better what is it. Am I right?
I recommend for any string added/modified adding a description that will be seen for translators:
```json
"ledStripLarsonOverlay": {
        "message": "Larson scanner",
        "description": "One of the modes for the LED strip feature"
},
```

Reply:
Yeah, it describes the function better, I think they leaved this translation key because legacy support.